Neos.ContentRepository.BehavioralTests/Classes/TestSuite/Behavior/ContentRepositoryInternalsAccessor.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neos\ContentRepository\BehavioralTests\TestSuite\Behavior;
+
+use Neos\ContentRepository\Core\Factory\ContentRepositoryServiceFactoryDependencies;
+use Neos\ContentRepository\Core\Factory\ContentRepositoryServiceFactoryInterface;
+use Neos\ContentRepository\Core\Factory\ContentRepositoryServiceInterface;
+
+class ContentRepositoryInternalsAccessor implements ContentRepositoryServiceFactoryInterface
+{
+    public ContentRepositoryServiceFactoryDependencies $spiedInternals;
```

Comment:
Suggestion: expose (just) the required dependencies as explicit public readonly fields